source/developer/contribution-guide.md

```diff
@@ -56,8 +56,9 @@ Before submitting a pull request (PR), check that:
 When submitting a PR, check that:  
 
 1. PR is submitted against `master`
-2. PR title begins with the Jira Ticket ID (eg `PLT-394:`, [see examples](https://github.com/mattermost/platform/pulls?q=is%3Apr+is%3Aclosed))
+2. PR title begins with the Jira Ticket ID (eg `PLT-394:`, [see examples](https://github.com/mattermost/mattermost-server/pulls?q=is%3Apr+is%3Aclosed))
 3. PR comment describes the changes and how the feature is supposed to work
+4. Multiple dependant PRs in multiple repositories must include the dependency between them in the description.
```

Comment:
I included it by myself, because have sense after the repository split.

Reply:
Thanks @jespino :)  Makes sense.